apps/docs/components/Copy/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useRef, useState } from "react";
+import CopyIcon from "./CopyIcon";
+import TickIcon from "./TickIcon";
+
+export default function Copy() {
+  const [copySuccess, setCopySuccess] = useState(false);
+  const buttonRef = useRef<HTMLButtonElement>(null);
+
+  useEffect(() => {
+    if (copySuccess) {
+      setTimeout(() => {
+        setCopySuccess(false);
+      }, 3000);
+    }
+  }, [copySuccess]);
+
+  function copyToClipboard(e: any) {
```

Comment:
Copy component should be generic. 
```<CopyToClipboard text={code} />```

Reply:
You can use navigator.clipboard.writeText().

Reply:
Okay, yes. I thought about that approach, but ended up taking this.
Will change it to make it a generic component. Thanks for the input. 👍🏻

Reply:
@ankitchouhan1020 Resolved this. Please have a look.